src/java/picard/analysis/CollectGcBiasMetrics.java

```diff
@@ -172,64 +153,40 @@ protected void finish() {
 
         final NumberFormat fmt = NumberFormat.getIntegerInstance();
         fmt.setGroupingUsed(true);
-        RExecutor.executeFromClasspath(R_SCRIPT,
+        /*RExecutor.executeFromClasspath(R_SCRIPT,
                 OUTPUT.getAbsolutePath(),
                 SUMMARY_OUTPUT.getAbsolutePath(),
                 CHART_OUTPUT.getAbsolutePath(),
-                String.valueOf(WINDOW_SIZE));
+                String.valueOf(WINDOW_SIZE));*/
```

Comment:
AGH yes sorry. The R script doesn't work with my intelliJ tests for some reason but works in tests when it's actually in github. I'll fix it now.